modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -33,34 +53,94 @@ export default function EditObjectFolder({
 }: EditObjectFolder) {
 	const [
 		{
+			baseResourceURL,
+			deleteObjectDefinition,
+			editObjectDefinitionURL,
 			elements,
+			modelBuilderModals,
 			objectDefinitionsStorageTypes,
 			objectFolderName,
 			rightSidebarType,
+			selectedObjectDefinitionNode,
 			selectedObjectFolder,
 			showChangesSaved,
 		},
 		dispatch,
 	] = useObjectFolderContext();
 
+	const [
+		objectRelationshipParameterRequired,
+		setObjectRelationshipParameterRequired,
+	] = useState(false);
+
+	const selectedObjectDefinitionNodeERC = selectedObjectDefinitionNode?.data
+		?.externalReferenceCode as string;
```

Comment:
Hey @Artartur, can you use the `selectedObjectDefinitionNode?.data?.externalReferenceCode` directly

Reply:
Done!